docs/src/dev-playground.md

```
@@ -0,0 +1,138 @@
+```bash
+kubectl create namespace elasti
+helm template elasti ./charts/elasti -n elasti -f ./playground/infra/elasti-demo-values.yaml | kubectl apply -f -
+```
+
+If you want to enable monitoring, please make `enableMonitoring` true in the values file.
+
+## 8. Create ElastiService Resource
+
+Using the [ElastiService Defination](./configure-elastiservice.md#configure-elastiservice), create a manifest file for your service and apply it. For demo, we use the below manifest.
+
+```bash
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)